index.html

```diff
@@ -12,39 +12,36 @@
 <body>
   <!-- Section 1: white portion/racoon -->
   <section>
-    <hr class="line"></hr>
-    <article id="racoonsection">
+    <hr class="line">
```

Comment:
Go ahead and remove these `hr`s as well and I'm not sure we need the `section` and the `article`. I'd remove the `articles` and put the class and id on the parent `section`.